concepticondata/conceptlists/Oliveira-2014-517.tsv

```diff
@@ -0,0 +1,518 @@
+Oliveira-2014-517-190	190	ferver	to boil		792	BOIL (OF LIQUID)		§187
+Oliveira-2014-517-191	191	mandíbula	jaw		798	JAW		§188
+Oliveira-2014-517-192	192	irmão da mãe	maternal aunt		2693	MATERNAL AUNT (MOTHER'S SISTER)		§189
+Oliveira-2014-517-193	193	vaga-lume	firefly		206	FIREFLY		§190
+Oliveira-2014-517-194	194	irmão da mãe	maternal uncle		2692	MATERNAL UNCLE (MOTHER'S BROTHER)		§191
+Oliveira-2014-517-195	195	ingerir alimentos moles	ingest soft food					§192
+Oliveira-2014-517-196	196	ingerir alimentos moles	ingest soft food					§193
+Oliveira-2014-517-197	197	nambu	tinamou		3921	TINAMOU		§194
+Oliveira-2014-517-198	198	esp. de fungo	species of funghi					§195
+Oliveira-2014-517-199	199	pus	pus		1808	PUS		§196
+Oliveira-2014-517-200	200	valor	value					§197
+Oliveira-2014-517-201	201	cinza, acinzentado	ash, gray		646	ASH		§198
+Oliveira-2014-517-202	202	cedro	cedrela odorata		3929	BRAZILIAN CEDAR		§199
+Oliveira-2014-517-203	203	cujubim	cujubim					§200
+Oliveira-2014-517-204	204	sufixo verbal causativo	verbal causative suffix					§201
+Oliveira-2014-517-205	205	morro, colina	hill		614	HILL		§202
+Oliveira-2014-517-206	206	esp. de peixe	species of fish					§203
+Oliveira-2014-517-207	207	frio	cold		1287	COLD		§204
+Oliveira-2014-517-208	208	varrer	to sweep		230	SWEEP		§205
+Oliveira-2014-517-209	209	chifre de animal	animal horn		1393	HORN (ANATOMY)		§206
+Oliveira-2014-517-210	210	terra	earth		1228	EARTH (SOIL)		§207
+Oliveira-2014-517-211	211	na terra, por terra	on earth					§208
+Oliveira-2014-517-212	212	enterrar	to bury		1719	BURY		§209a
+Oliveira-2014-517-213	213	faixa para a cabeça, chapéu	headbelt, hat		2130	HEADGEAR		§209
+Oliveira-2014-517-214	214	rato	rat		1490	RAT		§210
+Oliveira-2014-517-215	215	pedra, rocha	stone		857	STONE		§211
+Oliveira-2014-517-216	216	piranha (esp. de peixe)	piranha		317	PIRANHA		§212
+Oliveira-2014-517-217	217	esperar	to wait		1590	WAIT (FOR)		§213
+Oliveira-2014-517-218	218	sobre, encima, acima	above		1741	ABOVE		§214
+Oliveira-2014-517-219	219	metal	metal		2244	METAL		§215
+Oliveira-2014-517-220	220	banana	banana		868	BANANA		§216
+Oliveira-2014-517-221	221	sentir saudade	to feel longing		2328	MISS (SOMEBODY)		§217
+Oliveira-2014-517-222	222	subir (um morro)	ascend		705	GO UP (ASCEND)		§218
+Oliveira-2014-517-223	223	camarão	shrimp		2349	SHRIMP		§219
+Oliveira-2014-517-224	224	cabeça	head		1256	HEAD		§220
+Oliveira-2014-517-225	225	barro, poeira	clay, dust				CLAY//DUST	§221
+Oliveira-2014-517-226	226	esp. de planta	species of plant					§222
+Oliveira-2014-517-227	227	cutia (esp. de roedor)	cutia (species of rodent)		313	AGOUTI		§223
+Oliveira-2014-517-228	228	esp. de cabaça	species of gourd					§224
+Oliveira-2014-517-229	229	areia	sand		671	SAND		§225
+Oliveira-2014-517-230	230	pedra	stone		857	STONE		§226
+Oliveira-2014-517-231	231	urucum (a árvore e o seu fruto)	achiote		349	ACHIOTE		§227
+Oliveira-2014-517-232	232	encima, cume	atop		57	ATOP		§228
```

Comment:
Include also the noun translation; it should be peak

Reply:
Following this, I unmapped the entry